src/main/java/com/hermes/recruitment/service/CandidatService.java

```diff
@@ -0,0 +1,44 @@
+package com.hermes.recruitment.service;
+
+import com.hermes.recruitment.model.Candidat;
+import com.hermes.recruitment.model.dto.CandidatDTO;
+import com.hermes.recruitment.repository.ICandidatRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class CandidatService {
+
+    private final ICandidatRepository repository;
+
+    public void save(CandidatDTO candidatDTO) {
+        Candidat candidat = new Candidat(candidatDTO.getSurname(), candidatDTO.getFirstname(), candidatDTO.getMail(), candidatDTO.getPhone(), candidatDTO.getDepartment(), candidatDTO.getMotivation());

```

Comment:
here we can use a separate mapper object or bean; it would make the code cleaner